cms/static/js/certificates/spec/views/certificate_details_spec.js

```diff
@@ -133,7 +138,16 @@ function(_, Course, CertificatesCollection, CertificateModel, CertificateDetails
                 expect(this.view.$('.edit')).toExist();
             });
 
-            it('should change to "edit" mode when clicking the Edit button', function(){
+            it('should change to "edit" mode when clicking the Edit button and confirming the prompt', function(){
+                expect(this.view.$('.action-edit .edit')).toExist();
+                var promptSpy = ViewHelpers.createPromptSpy();
+                this.view.$('.action-edit .edit').click();
+                verifyAndConfirmPrompt(promptSpy, gettext('Edit this certificate?'));
```

Comment:
Nit: no need to use "gettext" within a test. We especially don't want to use it for test fixtures or anything that isn't already translated in the platform, since that causes unnecessary translation effort (not the case here).